scripts/IA/consume_files.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import os
+import argparse
+from zipfile import ZipFile
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+	'-g',
+	'--guid',
+	help='This is the GUID of the target node on the OSF'
+)
+parser.add_argument(
+	'-d',
+	'--directory',
+	help='This is the target Directory for the project and its files'
+)
+
+def main(default_args=True):
+	if (default_args):
+		args = parser.parse_args(['--guid', 'default', '--directory', 'default'])
+	else:
+		args = parser.parse_args()
+	args = parser.parse_args()
+	guid = args.guid
+	directory = args.directory
+
+	if not guid:
+		raise ValueError('Project GUID must be specified! Use -g')
+	if not directory:
+		# Setting default to current directory
+		directory = '.'
+
+	zip_url = 'https://files.osf.io/v1/resources/xxxxx/providers/osfstorage/?zip='
```

Comment:
This `xxxxx` shouldn't work as a replacement value.

Reply:
Oops. Forgot to fix this. Thanks for the heads up 👍